src/cdk/package.json

```diff
@@ -1,12 +1,11 @@
 {
   "name": "@angular/cdk",
-  "version": "0.0.0-PLACEHOLDER",
+  "version": "2.0.0-beta.8",
```

Comment:
Are you sure about this one? The only thing that's reasonable (why it not updated the placeholder) is actually the peer dependency from Material.